OpenRA.Test/OpenRA.Game/SyncImplTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+
+namespace OpenRA.Test
+{
+	[TestFixture]
+	class SyncImplTest
+	{
+		static readonly IEnumerable<Type> AllTypes =
+			Directory.EnumerateFiles("mods", "OpenRA.Mods.*.dll", SearchOption.AllDirectories)
```

Comment:
This isn't very nice: mods are free to load dlls from wherever they want (this becomes even more important after #8992).